hole/jacobi-symbol.go

```diff
@@ -0,0 +1,48 @@
+package hole
+
+import (
+	"crypto/rand"
```

Comment:
Do we need cryptographic random numbers? I would've assumed that just `math/rand` would suffice.

Reply:
I imported that module for random primes. I used it for the regular random numbers too for convinience but I will change that.